From bcf4d44941c66a274f0346825d435561ef958f4b Mon Sep 17 00:00:00 2001
From: Jing Hua <jinghua@marvell.com>
Date: Sun, 6 Sep 2015 17:06:05 +0800
Subject: [PATCH 0413/1240] fix: tools: arlp: update TIM to keep CPU1 in
 HeldInRst state.

To test PSCI feature and bring-up CPU1 in kernel, need to keep
CPU1 in HeldInRst state.
Got this package from Helen Li.
Tested on Palladium.

Change-Id: I19a08304d2617781a688d696084c665ae683767e
Signed-off-by: Jing Hua <jinghua@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/23427
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
---
 tools/wtp/u-boot-ntim.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/tools/wtp/u-boot-ntim.txt b/tools/wtp/u-boot-ntim.txt
index 6fbe556..a917860 100644
--- a/tools/wtp/u-boot-ntim.txt
+++ b/tools/wtp/u-boot-ntim.txt
@@ -33,8 +33,8 @@ Reserved Data:
 0x00000002
 0x00000000
 0x00000000
-0x00000001
-0x10000081
+0x00000002
+0x1fff0081
 0x5465726D
 0x00000008
 
@@ -138,6 +138,7 @@ WRITE: 0xC0001020 0x40000000
 WRITE: 0xC0001020 0x80000000
 WRITE: 0xC0000020 0x11000001
 WAIT_FOR_BIT_SET: 0xC0000008 0x00000001 0x00001000
+WRITE: 0x60030000 0x01234567
 End Instructions:
 End DDR Initialization:
 End Extended Reserved Data:
-- 
1.9.1

